modules/update-upgrading-web.adoc

```diff
@@ -65,7 +65,7 @@ The Input channel
 +
 [NOTE]
 ====
-If you are update your cluster to the next minor version, like version 4.y to 4.(y+1), it is recommended to confirm your nodes are updated before deploying workloads that rely on a new feature. Any pools with worker nodes that are not yet updated are displayed on the *Cluster Settings* page.
+If you are updating your cluster to the next minor version, for example from version 4.y to 4.(y+1), confirm that your nodes are updated before deploying workloads that rely on a new feature. Any pools with worker nodes that are not yet updated are displayed on the *Cluster Settings* page.
```

Comment:
I don't think a change is necessarily _required_ here, but "4.y to 4.(y+1)" reads very awkwardly to me. If we're already using an example "4" as a major version number, couldn't we provide a completely fictional number in the minor version as well instead of having to construct a "y+1" example?

Reply:
One one hand, keeping the minor version as a variable prevents the example from ever being awkwardly irrelevant and implying that we don't maintain the docs (e.g. mentioning 4.9 to 4.10 in a 4.22 doc), and prevents the tech debt to update the versions to more relevant numbers.

On the other hand, there's definitely merit to the idea that if we're mentioning something as an example, we may as well use an _actual_ example, and the current phrasing is certainly more clunky than if we were to use real numbers. I think I'll change it to real versions and plan to update this, along with other references like this in the docs, periodically.